code/datums/status_effects/debuffs.dm

```diff
@@ -967,3 +967,81 @@
 /datum/status_effect/cloudstruck/Destroy()
 	. = ..()
 	QDEL_NULL(mob_overlay)
+
+//Deals with ants covering someone.
+/datum/status_effect/ants
+	id = "ants"
+	tick_interval = 1 SECONDS
+	status_type = STATUS_EFFECT_REFRESH
+	alert_type = /atom/movable/screen/alert/status_effect/ants
+	duration = 5 MINUTES //Keeping the normal timer makes sure people can't somehow dump 500+ ants on someone so they stay there for like 30 minutes
+	examine_text = "<span class='warning'>SUBJECTPRONOUN is covered in ants!</span>"
+	var/ants_remaining = 0 //Will act as the main timer as well as changing how much damage the ants do.
+
+/datum/status_effect/ants/on_creation(mob/living/new_owner, amount_left)
+	if(isnum(amount_left) && ants_remaining == 0)
+		to_chat(new_owner, "<span class='userdanger'>You're covered in ants!</span>")
+		ants_remaining = amount_left
+		RegisterSignal(new_owner, COMSIG_COMPONENT_CLEAN_ACT, .proc/ants_washed)
+	else if(ants_remaining >= 0)
+		to_chat(new_owner, "<span class='userdanger'>You're covered in MORE ants!</span>")
+		if(prob(1)) //Extremely rare
+			new_owner.say("THIS SITUATION HAS BEEN MADE WORSE BY THE ADDITION OF YET MORE ANTS!!", forced = /datum/status_effect/ants)
+		ants_remaining += amount_left
+	. = ..()
+
+/datum/status_effect/ants/on_remove()
+	ants_remaining = 0
+	to_chat(owner, "<span class='notice'>All of the ants are off of your body!</span>")
+	UnregisterSignal(owner, COMSIG_COMPONENT_CLEAN_ACT, .proc/ants_washed)
+	. = ..()
+
+/datum/status_effect/ants/proc/ants_washed()
+	SIGNAL_HANDLER
+	owner.remove_status_effect(STATUS_EFFECT_ANTS)
+	return COMPONENT_CLEANED
+
+/datum/status_effect/ants/tick(delta_time)
+	var/mob/living/carbon/human/victim = owner
+	victim.adjustBruteLoss(max(0.2, round((ants_remaining * 0.008),0.1))) //Roughly 20 brute over 50 seconds.
+	if(!victim.stat) //Makes sure people don't scratch at themselves while they're dead
```

Comment:
Will they continue to stack bruteloss if dead? should the ants just give up if you die?

Reply:
Good point, I should probably do that really quick.